pkg/types/broker.go

```diff
@@ -46,9 +49,13 @@ func (b *Broker) IsEmpty() bool {
 // TableData returns the data to populate a table
 func (b *Broker) TableData() *TableData {
 	result := &TableData{}
-	result.Headers = []string{"ID", "Name", "URL", "Description", "Created", "Updated", "Labels"}
+	result.Headers = []string{"ID", "Name", "URL", "Description", "Created", "Updated", "Labels", "Last Op"}
 
-	row := []string{b.ID, b.Name, b.URL, b.Description, b.Created, b.Updated, formatLabels(b.Labels)}
+	lastState := "-"
+	if b.LastOperation != nil {
+		lastState = string(b.LastOperation.State)
+	}
+	row := []string{b.ID, b.Name, b.URL, b.Description, b.Created, b.Updated, formatLabels(b.Labels), lastState}
```

Comment:
Does it make sense to also add the type of the last operation?

Reply:
with -o json you can see the whole last operation object, so for now I will leave it as is